onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -40,7 +40,7 @@
 TFloatUnlessFloat32 = TypeVar("TFloatUnlessFloat32", bound=Union[BFLOAT16, FLOAT16, DOUBLE])
 
 
-@torch_op("aten::aten_adaptive_avg_pool1d", traceable=True)
+@torch_op("aten::adaptive_avg_pool1d", traceable=True)
```

Comment:
Is there decomp for these?

Reply:
Yes, it's forced to be decomposed, just like upsampling.

Reply:
If that's the case, we may be better off forcing the decomp since we don't have full support for adaptive pooling

Reply:
It usually starts with `@register_decomposition()`, but for this one, I haven't found it yet.